pkg/errors/errors.go

```diff
@@ -326,6 +348,12 @@ func checkError(wrappedError, err error) bool {
 		return true
 	}
 
+	wrappedErrorLowercase := strings.ToLower(wrappedError.Error())
```

Comment:
Ideally, I would like to use the lowercase strings and club the 2 strings.Contains calls. Please let me know your thoughts on this @ggaurav10, @madhanrm , @in3xes